src/app/(main)/my-voyage/[teamId]/ideation/finalize/components/FinalizeIdeationItem.tsx

```diff
@@ -0,0 +1,53 @@
+import { Dispatch, SetStateAction } from "react";
+import { CheckCircleIcon } from "@heroicons/react/24/outline";
+import Button from "@/components/Button";
+import Avatar from "@/components/avatar/Avatar";
+import AvatarGroup from "@/components/avatar/AvatarGroup";
+import { ProjectIdeaVotes } from "@/store/features/ideation/ideationSlice";
+
+interface FinalizeIdeationItemProps {
+  title: string;
+  projectIdeaVotes: ProjectIdeaVotes[];
+  finalizedIdeation: string;
+  setFinalizedIdeation: Dispatch<SetStateAction<string>>;
+}
+
+export default function FinalizeIdeationItem({
+  title,
+  projectIdeaVotes,
+  finalizedIdeation,
+  setFinalizedIdeation,
+}: FinalizeIdeationItemProps) {
+  function handleClick() {
+    setFinalizedIdeation(title);
+  }
+
+  return (
+    <Button
+      variant={finalizedIdeation === title ? "primary" : "outline"}
+      className="w-[800px] gap-x-0"
```

Comment:
To make it responsive: instead of `w-[800px]` you can use `max-w-[800px] w-full`